### PR TITLE
Post announcing release of the book

### PR DESCRIPTION
**Pull request by katyhuff**
The book hath been released. I can haz blog post tomorrow morning?

Proofreading welcome. And, @scopatz may want to make a change or two in the next few hours... otherwise.. it should be good to go for the morning. 

:D

### CODE REVIEW

**Comment by r-gaia-cs** [comment]
>  I can haz blog post tomorrow morning?

:+1:

File: blog/2015/07/effective-physics-book.html [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+<div align="center">
+  <p><strong>The Cover</strong></p>
+  <img src="http://physics.codes/images/ecip.jpg" class="responsive" />
+</div>
+<br/>
+<p>
+In addition to being adorned by the bobtail squid, the book is emblazoned with
+the kind words of two of our mentors. First and foremost, upon handling the book, the
+reader will encounter a quotation from Fernando Perez on the back cover.
+Additionally, the first few pages contain a foreword by Greg Wilson himself.
```
Comment (gvwilson) [inline blog/2015/07/effective-physics-book.html]: Please remove the "himself" :-)

File: blog/2015/07/effective-physics-book.html [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+<b>Anthony and I are extremely humbled and proud to have the kind support of
+these two extraordinary individuals, without whom this book could not have
+existed.</b> To read what they said, you'll have to get your hands on a
+book.  All I'll say here is that both were kinder than we could have dreamed.
+</p>
+<h2>What's In It?</h2>
+<p>Since different scientists struggle in different ways at each stage of the
+research game, we wanted to cover the whole playbook of scientific computing
+best practices. Thus, we formed this book around the idea of a soup-to-nuts,
+cradle-to-grave, idea-to-publication scientific research workflow, split into four
```
Comment (gvwilson) [inline blog/2015/07/effective-physics-book.html]: I'd take out "cradle-to-grave"

File: blog/2015/07/effective-physics-book.html [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+<li>Chapter 20 - Publication</li>
+<li>Chapter 21 - Collaboration</li>
+<li>Chapter 22 - Licenses, Ownership, and Copyright</li>
+<li>Chapter 23 - Further Musings on Computational Physics</li>
+</ul>
+
+
+<h2>Where To Get It</h2>
+
+<ul>
```
Comment (gvwilson) [inline blog/2015/07/effective-physics-book.html]: Why lay this out as a list rather than a paragraph?

**Comment by gvwilson** [comment]
:+1:

**Comment by katyhuff** [comment]
@gvwilson thanks for the suggestions. All implemented!! Ready to merge in the morning. I'll do it if you don't have a chance. Thanks!

**Pull request merged**